frontend/app/actions/catActions.js

```diff
@@ -0,0 +1,24 @@
+import { RSAA } from 'redux-api-middleware'
+
+export const REQUEST = 'RANDOM_CAT_REQUEST'
+export const SUCCESS = 'RANDOM_CAT_SUCCESS'
+export const FAILURE = 'RANDOM_CAT_FAILURE'
+
```

Comment:
I will create a separate module for action type constants when this grows. Just left them here for now.